content/docs/installation/best-practice.md

```diff
@@ -22,6 +22,109 @@ are designed for backwards compatibility rather than for best practice or maximu
+These are two reasons why you should consider isolating the cert-manager components from
+other less privileged workloads.
+For example, if an untrusted or malicious workload runs on the same Node as the cert-manager controller,
+and somehow gains root access to the underlying node,
+it may be able to read the private keys in Secrets that the controller has cached in memory.
+
+You can mitigate this risk by running cert-manager on nodes that are reserved for trusted platform operators.
+
+The Helm chart for cert-manager has parameters to configure the Pod `tolerations` and `nodeSelector` for each component.
+The exact values of these parameters will depend on your particular cluster.
+
+### Example
+
+This example demonstrates how to use:
+`taints` to *repel* non-platform Pods from Nodes which you have reserved for your platform's control-plane,
+`tolerations` to *allow* cert-manager Pods to run on those Nodes, and
+`nodeSelector` to *place* the cert-manager Pods on those Nodes.
+
+Label the Nodes:
+
+```bash
+kubectl label node ... node-restriction.kubernetes.io/reserved-for=platform
+```
+
+Taint the Nodes:
+
+```bash
+kubectl taint node ... node-restriction.kubernetes.io/reserved-for=platform:NoExecute
+```
+
+Then install cert-manager using the following Helm chart values:
+
+```yaml
+nodeSelector:
+  kubernetes.io/os: linux
+  node-restriction.kubernetes.io/reserved-for: platform
+tolerations:
+- key: node-restriction.kubernetes.io/reserved-for
+  operator: Equal
+  value: platform
+
+webhook:
+  nodeSelector:
+    kubernetes.io/os: linux
+    node-restriction.kubernetes.io/reserved-for: platform
+  tolerations:
+  - key: node-restriction.kubernetes.io/reserved-for
+    operator: Equal
+    value: platform
+
+cainjector:
+  nodeSelector:
+    kubernetes.io/os: linux
+    node-restriction.kubernetes.io/reserved-for: platform
+  tolerations:
+  - key: node-restriction.kubernetes.io/reserved-for
+    operator: Equal
+    value: platform
+
+startupapicheck:
+  nodeSelector:
+    kubernetes.io/os: linux
+    node-restriction.kubernetes.io/reserved-for: platform
+  tolerations:
+  - key: node-restriction.kubernetes.io/reserved-for
+    operator: Equal
+    value: platform
+```
+
+> ℹ️ This example uses `nodeSelector` to *place* the Pods but you could also use `affinity.nodeAffinity`.
+> `nodeSelector` is chosen here because it has a simpler syntax.
+> Read [Assigning Pods to Nodes](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/) to learn more.
+>
+> ℹ️ The default `nodeSelector` value `kubernetes.io/os: linux` [avoids placing cert-manager Pods on Windows nodes in a mixed OS cluster](https://github.com/cert-manager/cert-manager/pull/3605),
+> so that must be explicitly included here too.
+>
+> 📖 Read more about [Taints and Tolerations](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/)
+> in the [Kubernetes documentation](https://kubernetes.io/docs/).
+>
```

Comment:
It would be more convenient for readers if this link were introduced earlier, right when these concepts are first mentioned.